tensorflow/python/util/nest.py

```diff
@@ -137,7 +145,7 @@ def _sequence_like(instance, args):
     result = dict(zip(_sorted(instance), args))
     instance_type = type(instance)
     if instance_type == _collections.defaultdict:
-      d = _collections.defaultdict(instance.default_factory)
+      d = instance_type(_collections.defaultdict(instance.default_factory))
       for key in instance:
         d[key] = result[key]
       return d
```

Comment:
Don't forget to update line 153: we want to rebuild the mapping with a for loop, not directly using the constructor.

Reply:
Line 153 is using a for loop to return a mapping between the result and the key.

Reply:
I'm seeing just `return instance_type((key, result[key]) for key in instance)`

Reply:
I've committed with the requested changes.